Formula/mas.rb

```diff
@@ -14,11 +14,34 @@ class Mas < Formula
   depends_on :xcode => ["9.0", :build]
 
   def install
-    xcodebuild "-project", "mas-cli.xcodeproj",
+    # Pre-install a shallow copy of the CocoaPods master repo
+    system "git", "clone", "--depth", "1", "https://github.com/CocoaPods/Specs.git", File.expand_path("~/.cocoapods/repos/master")
```

Comment:
Can we add this as a resource and then stage the resource instead of using `git clone` directly?

Reply:
That's a good idea. 💡